src/MAVN.Service.PartnerManagement.Domain/Constants/GeohashLevelsSmallerSidesInKm.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace MAVN.Service.PartnerManagement.Domain.Constants
+{
+    /// <summary>
+    /// Holds values of the smaller sized side of the geohash rectangle
+    /// More info about geohash https://www.elastic.co/guide/en/elasticsearch/guide/current/geohashes.html#geohashes
+    /// </summary>
+    public class GeohashLevelsSmallerSidesInKm
```

Comment:
Let's make this a list of values, so iterating through it would be easier in the using code